### PR TITLE
Fixing a couple issues filed against master.

### PR DESCRIPTION
**Pull request by Sfshaza**
@brianegan  @mogest  (Brian, Ian thought this change was reasonable, so I proceeded.)

Please review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,14 +190,14 @@ it with triple backticks followed by the language.
...
-      Widget build(BuildContext context) {
-        return new Container();
-      }
-    }
-	```
+```dart
+class ExampleWidget extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new Container();
```
Comment (chalin) [inline README.md]: We can drop `new` now.

File: src/cookbook/networking/fetch-data.md [hunk 2]
```diff
@@ -145,6 +145,60 @@ FutureBuilder<Post>(
 );
 ```
 
+## 5. Moving the fetch call out of the `build` method
+
+Although it's convenient, it's not recommended to put a call to an API in your
+`build` method.
```
Comment (chalin) [inline src/cookbook/networking/fetch-data.md]: Per writing guidelines method names should be followed by parentheses: `build()`. (This should be applied on this line and a few below here.)

**Comment by brianegan** [comment]
Hi @Sfshaza! Yep, this looks like a great change. My initial goal was to keep that section of the recipe short based on earlier feedback, but I think it's really important to provide that information. It's a common mistake.

File: README.md (hunk 1 above)
Comment (Sfshaza) [reply README.md]: Good catch!

File: src/cookbook/networking/fetch-data.md (hunk 2 above)
Comment (Sfshaza) [reply src/cookbook/networking/fetch-data.md]: Yeah, we're kinda sorta consistent about this.  :)  Fixed.

**Comment by Sfshaza** [comment]
Thanks for your quick reviews, @brianegan and @chalin! Feedback incorporated.

**Comment by Sfshaza** [comment]
Sorry about that, @mogest. We are in a fairly peculiar state right now, working on getting out a new version of the website. We aren't making changes to master unless they are P0. If you go to one of the flutter.io pages, and hover over the Edit source text, it will say that we aren't accepting changes at this time. You may have noticed that I put those edits into a branch that isn't master.

Ordinarily, I'd be thrilled to accept a community-provided PR, so long as it meets requirements. (It can be difficult when someone submits a PR involving many files, because I have no way to cherry pick and not all edits are valid.) This situation was complicated by the fact that @brianegan (the cookbook author) is a contractor and was on a sabbatical, delaying his review of your original PR.

I'm sorry that the timing worked out this way. Once the new website is out, I'd be happy to return to accepting community PRs. I am a lone Flutter writer, so I welcome the help.  :)

**Pull request merged**